packages/app/src/explorer/EntityItem.tsx

```diff
@@ -55,6 +57,37 @@ function EntityItem(props: Props) {
 
           onSelect(path);
         }}
+        onKeyDown={(e) => {
+          switch (e.key) {
+            case 'ArrowDown':
+              focusNext(e);
+              return;
+
+            case 'ArrowUp':
+              focusPrevious(e);
+              return;
+
+            case 'ArrowLeft':
+              if (isGroup(entity) && isExpanded) {
+                toggleExpanded(false);
+                e.preventDefault();
+              } else {
+                focusParent(e, path);
+              }
+              return;
+
+            case 'ArrowRight':
+              if (!isGroup(entity)) {
+                return;
+              }
+              if (isExpanded) {
+                focusNext(e);
+              } else {
+                toggleExpanded(true);
+                e.preventDefault();
```

Comment:
Move this before the `if/else` ?

Reply:
I wish to let `focusNext` handle `preventDefault` in the `if` branch. 

For example, if we reached the end of the tree and we press `Down`, the default behaviour (scrolling the explorer) should be applied.

Reply:
Good call 💯